frontend/apps/marketing/src/config/studio.ts

```diff
@@ -0,0 +1,14 @@
+import {getStage} from './stage';
+
+export function getStudioUrl() {
```

Comment:
Can we toss in a basic unit test for this? Co-pilot generated would be fine

Reply:
Added!